src/AppBundle/Controller/CommitteeController.php

```diff
@@ -49,11 +53,35 @@ public function showAction(Committee $committee, Request $request): Response
             'committee' => $committee,
             'committee_members_count' => $committeeManager->getMembersCount($committee),
             'committee_hosts' => $committeeManager->getCommitteeHosts($committee),
+            'committee_timeline' => $committeeManager->getTimeline($committee, $this->isGranted('ROLE_ADHERENT')),
             'form' => $form ? $form->createView() : null,
         ]);
     }
 
     /**
+     * @Route("/timeline", name="app_committee_timeline")
+     * @Method("GET")
+     */
+    public function timelineAction(Request $request, Committee $committee): Response
+    {
+        if (!$request->isXmlHttpRequest()) {
+            throw new BadRequestHttpException('Not an XMLHttpRequest!');
+        }
+
+        $timeline = $this->get('app.committee_manager')->getTimeline(
+            $committee,
+            $this->isGranted('ROLE_ADHERENT'),
```

Comment:
The committee manager should not be responsible, we should use a `CommitteeFeedManager` instead of the handler with the authorization checker as dependency. Next PR though.